src/main/java/seedu/storemando/ui/MainWindow.java

```diff
@@ -187,11 +194,14 @@ public ItemListPanel getItemListPanel() {
     }
 
     /**
-     * Updates the expiring item table.
+     * Updates the both the expiring and expired item table.
      */
     private void updateExpiringItemTablePanel() {
-        tablePanel = new TablePanel(logic.getFilteredItemList());
-        tablePanelPlaceholder.getChildren().add(tablePanel.getRoot());
+        expiredTablePanel = new ExpiredTablePanel(logic.getFilteredItemList());
```

Comment:
Should this be retrieving the StoreMando item list instead of the filtered list? Otherwise, the table would be for filtered list of items instead of all the items in the list after every user command.

Reply:
Yes, you are right. I made forgot to change this part.